src/words/EditButton.js

```diff
@@ -39,12 +48,31 @@ export default function EditButton({
       bookmark.context,
     );
     api.updateBookmark(bookmark.id, newWord, newTranslation, newContext);
+    if (newFitForStudy) {
```

Comment:
The state that determines the checked status is the fitForStudy, if the user then unsets / sets it it will override the fitForStudy, thus making it different. 

In other words, the checkbox is not a display if the user has preferred that word, but rather of the current word status. It could be from Zeeguu or their own preferenes, we don't get about which source it is.